### PR TITLE
created search box and placeholder for results

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
all tasks complete but still some code duplication in script - will need to refactor during the week

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Pathway+Gothic+One" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>YETFLIX</title>
+</head>
+
+<header>
+    <div class="top-menu">
+        <i class="fas fa-heart"></i>
+        <a href='#' class="logo">YETFLIX</a>
```
Comment (dmitrigrabov) [inline index.html]: link here is not really needed as it does not go anywhere

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <i class="fas fa-sign-out-alt"></i>
+    </div>
+    <div class='favs'></div>  
+</header>
+
+<body>
+
+<div class='search'>
```
Comment (dmitrigrabov) [inline index.html:24]: nice use of BEM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        </div>
+        <button class='search__button' type="submit"><i class="search__icon fa fa-search"></i></button>
+    </form>
+</div>
+
+<div class='results__wrapper'>
+    <p class='results__title'></p>
+    <div class='results'></div>
+    <nav class='nav'>
+            <a href="#" class="nav__prev">&#8249;</a>
```
Comment (dmitrigrabov) [inline index.html]: It might be better to use buttons here rather than links that don't go anywhere

File: index.js [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        getMoreResults();
+    }
+});
+
+//Event listeners: favourite checkboxes
+document.addEventListener('change', event => {
+    if (event.target.matches('.detail__checkbox')) {
+        const imdbID = event.target.getAttribute('data-id');
+        const title = event.target.parentNode.parentNode.parentNode.getAttribute('data-title');
+        if (event.target.checked) addToFavourites(imdbID, title);
```
Comment (dmitrigrabov) [inline index.js]: it's better practice to use curly brackets with if/else, otherwise it is slightly more difficult to read

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: changed logo to h1 element. thx

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html]: changed to buttons. thanks

File: index.js (hunk 4 above)
Comment (ghost) [reply index.js]: added curlies, thx

**Pull request closed without merging**